tools/pyclient/src/molgenis_emx2_pyclient/client.py

```diff
@@ -109,22 +109,6 @@ def signin(self, username: str, password: str):
                       f"\n{response_json.get('message')}"
             log.error(message)
             raise SigninError(message)
-
-    def signout(self):
-        """Signs the client out of the EMX2 server."""
-        response = self.session.post(
-            url=self.api_graphql,
-            json={'query': queries.signout()}
-        )        
-        
-        status = response.json().get('data', {}).get('signout', {}).get('status')
-        if status == 'SUCCESS':
-            print(f"User '{self.username}' is signed out of '{self.url}'.")
-            self.signin_status = 'signed out'
-        else:
-            print(f"Unable to sign out of {self.url}.")
-            message = response.json().get('errors')[0].get('message')
-            print(message)
```

Comment:
I'm worried about getting rid of the signout method. Why are the servers able to handle sessions that are not signed out now? I've had servers crash due to this before

Reply:
It appears that `signout` is back in graphql. I will bring back the signout method.